app.js

```diff
@@ -183,6 +183,13 @@ function renderChart() {
   });
 }
 
+function setAllProducts (){
+  let stringify = JSON.stringify(allProducts);
+  localStorage.setItem('All Products', stringify);
+
+  localStorage.getItem('All Products');
+  JSON.parse(stringify);
```

Comment:
This should probably be in a different function. Once you set the data, getting the data will get the data that you already have.